src/pages/DashboardPage.tsx

```diff
@@ -1,8 +1,74 @@
 import React from "react";
-import { Typography } from "@mui/material";
+import { Box, Container, Stack, Typography, useTheme } from "@mui/material";
+import { Link } from "react-router-dom";
+
+type ModulesProps = {
+  title: string;
+  description: string;
+  href: string;
+};
+
+const modules: ModulesProps[] = [
+  {
+    title: "Suppliers",
+    description: "Manage the suppliers for the system.",
+    href: "/suppliers",
+  },
+  {
+    title: "Users",
+    description: "Manage the users in the system.",
+    href: "/users",
+  },
+];
 
 const DashboardPage: React.FC = () => {
-  return <Typography variant="h4">Welcome to the Dashboard</Typography>;
+  const theme = useTheme();
+
+  return (
+    <Container sx={{ mt: 4 }}>
+      <Typography variant="h4" fontWeight={600} mb={4}>
+        Welcome to the Dashboard
+      </Typography>
+      <Stack
+        direction={{ xs: "column", md: "row" }}
+        spacing={4}
+        alignItems="stretch"
+      >
+        {modules.map((module) => (
+          <Box
+            key={module.title}
+            component={Link}
+            to={module.href}
+            sx={{
```

Comment:
Use mui styled for styling instead of inline styled and seperate styles in seperate file `style.tsx`
https://mui.com/system/styled/